apps/common-app/App.tsx

```diff
@@ -435,18 +262,15 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     justifyContent: 'space-between',
   },
-  buttonContent: {
+  settingsButton: {
     flex: 1,
+    paddingHorizontal: 20,
     flexDirection: 'row',
-    alignItems: 'center',
-    justifyContent: 'space-between',
-  },
-  autoOpenSetting: {
-    margin: 16,
-    borderRadius: 16,
+    margin: 12,
+    borderRadius: 10,
     backgroundColor: '#eef0ff',
-    paddingHorizontal: 16,
-    justifyContent: 'space-between',
+    padding: 10,
+    justifyContent: 'center',
```

Comment:
Can we stick to multiples of 4?

Also, the gap between the buttons is larger than the gap between the buttons and the screen edge.